ucsb-cs156-f23/team03-f23-6pm-1
# Erwan help request pages

Created the create, index, and edit pages for Help Requests along with their respective test cases. 

Also switched the solved entry in the form to a switch to better support booleans.

Index Page Storybook: https://ucsb-cs156-f23.github.io/team03-f23-6pm-1/prs/82/storybook/?path=/story/pages-helprequests-helprequestsindexpage--three-items-ordinary-user
Create Page Storybook: https://ucsb-cs156-f23.github.io/team03-f23-6pm-1/prs/82/storybook/?path=/story/pages-helprequests-helprequestscreatepage--default
Edit Page Storybook: https://ucsb-cs156-f23.github.io/team03-f23-6pm-1/prs/82/storybook/?path=/story/pages-helprequests-helprequestseditpage--default

Index Page:

<img width="1272" alt="Screenshot 2023-11-08 at 2 42 01 PM" src="https://github.com/ucsb-cs156-f23/team03-f23-6pm-1/assets/36494167/78309bdf-c304-4b2b-bf0b-6ca9f4753c97">

Create page:

<img width="1260" alt="Screenshot 2023-11-08 at 2 34 09 PM" src="https://github.com/ucsb-cs156-f23/team03-f23-6pm-1/assets/36494167/fc348360-d65d-4046-a327-0cd358ac536e">

Testing Procedure:
1. Fill in the form with the required information and hit submit. Make sure your new submission is in the database using the swagger endpoint.
2. Repeat the same procedure but hit cancel, make sure your new submission is not put in the database.
3. Repeat the same procedure but with incorrect information and or missing information, make sure that this submission is not entered into the database and you receive messages to fix the info.
4. Test to ensure that navigating to /helprequest/create brings you to this page.

Edit Page:

<img width="1260" alt="Screenshot 2023-11-08 at 2 36 55 PM" src="https://github.com/ucsb-cs156-f23/team03-f23-6pm-1/assets/36494167/4c3ecb56-8557-4e93-ba48-9ddd7d7d6bec">

Closes #49 
Closes #50 
Closes #51
